local-cli/start_packager.sh

```diff
@@ -0,0 +1,22 @@
+
+set -e
+
+HEADER="${SRCROOT}/Modules/RCTIPAddress.h"
+
+echo "// This file is generated by a build script." > $HEADER
+echo "// Updated on "`date` >> $HEADER
+echo "" >> $HEADER
+
+ip_address="`ifconfig en0 | grep "inet " | cut -d: -f2 | awk '{print $2}'`"
```

Comment:
@mroswald I just copied that line from a commit linked by [this comment](https://github.com/facebook/react-native/issues/6291#issuecomment-193621003). This has also been discussed in depth in #4245. It can't be `ipconfig getifaddr en0` because you might change networks often (eg: you go to a coffee shop, or you use ethernet at work). I doubt what I have is the perfect solution, though. As long as there is a way to override the automatic IP, I don't think it hurts to use the current solution and build upon it when deemed necessary.